src/api/app/views/webui2/webui/comment/_comment_field.html.haml

```diff
@@ -0,0 +1,7 @@
+= form_for(comment, action: 'webui2_create', method: :post, remote: true, html: { class: 'new-comment-form' }) do |f|
+  = hidden_field_tag :commentable_type, commentable.class
+  = hidden_field_tag :commentable_id, commentable.id
+  = f.hidden_field :parent_id, value: comment.parent_id
+  ~ f.text_area :body, rows: '4', placeholder: 'Add a new comment (markdown markup supported)', required: true,
+    class: 'w-100 mb-3 form-control comment-field'
+  = f.submit 'Add comment', class: 'btn btn-primary', data: { disable_with: 'Creating comment...' }
```

Comment:
[nitpick] _Creating_ still sounds to me not very user friendly. I would use _Sending_